INN/umbrella-inndev
# Update to inn.org homepage for https://github.com/INN/inn/issues/117

## Changes

This pull request makes the following changes:

- Updates the inn.org homepage in accordance with 

## Why



For #

## Testing/Questions

Features that this PR affects:

- inn.org submodules


Questions that need to be answered before merging:


Steps to test this PR:

1. visit https://inndev.staging.wpengine.com/, where this is already deployed

deploy steps:
1. merge to master,
2. deploy via git push
3. Check that the INN/inn submodel checked out, referencing that one other WPE ticket.
